cmd/bee/cmd/cmd.go

```diff
@@ -61,6 +61,7 @@ const (
 	optionNameSwapInitialDeposit         = "swap-initial-deposit"
 	optionNameSwapEnable                 = "swap-enable"
 	optionNameChequebookEnable           = "chequebook-enable"
+	optionNameStubChainBackend           = "stub-chain-backend" //TODO rename, add docs about no overlay,postage,accounting,settlements
```

Comment:
The name for the flag is very much up for discussion. We want it to convey that its purpose is to enable or disable the chain back-end.